nixos/modules/misc/ids.nix

```diff
@@ -354,6 +354,7 @@ in
       webdav = 322;
       pipewire = 323;
       rstudio-server = 324;
+      whoogle-search = 325;
```

Comment:
Can we use dynamic users?

Reply:
I will look into it, thanks for the tip!